staff/lab/last-lab

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import sys
+
+import ocflib.lab.stats
+from ocflib.infra.hosts import hostname_from_domain
+from tabulate import tabulate
+
+
+def hostname(s):
+    """Argparse custom type that appends ".ocf.berkeley.edu" to short hostnames."""
+    if '.' not in s:
+        return s + '.ocf.berkeley.edu'
+    else:
+        return s
+
+
+def get_credentials():
+    """Return dictionary of ocfstats-ro creds, if we can read them."""
+    return json.load(open('/etc/ocfstats-ro.json', 'r'))
```

Comment:
I'm assuming this is coming in a puppet PR?

Reply:
Also why not just make this a plain file with just a password instead of having to load it from JSON? Seems easier to me and less likely to break, and I don't see us changing the MySQL user all that often.

Reply:
I largely copied this code from the `paper` script.

Sure, we could make it a plain text file. I guess that invites the question of whether we should make `paper` do the same though...